app/views/works/_organization_input.html.erb

```diff
@@ -0,0 +1,7 @@
+<%# This is only for demo purposes before affiliation implementation %>
+<div data-controller="autocomplete" data-autocomplete-url-value="/ror" data-autocomplete-min-length-value="3" class="dropdown" role="combobox" aria-expanded="false" data-autocomplete-ready-value="true">
+  <input class="form-control" data-autocomplete-target="input" type="text" name="DEMO-ORGANIZATION" id="DEMO-ORGANIZATION" autocomplete="off" spellcheck="false">
+  <!-- This input is populated by autocomplete controller.-->
+  <input type="hidden" data-autocomplete-target="hidden" data-autocomplete-edit-target="value">
+  <ul class="list-group" data-autocomplete-target="results" hidden=""></ul>
```

Comment:
Is Siteimprove happy with these changes or do we need some aria labels?

Reply:
Since this is going to be thrown away, I didn't SiteImprove it. All this does is put a plain-old input on the page without even a label.